### ISSUE TITLE
[Bug]: Task failed: 'bool' object does not support item assignment

### ISSUE THREAD

**Issue by LeoPadre** [tubearchivist/tubearchivist#563]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [bug report](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#bug-report) section.

### Operating System

Synology

## Describe the bug
Error: Task failed: 'bool' object does not support item assignment

## Steps To Reproduce
When adding a playlist via the plus button on the playlists tab, I get an error and the playlist is not added. 
![Снимок экрана 2023-10-01 в 18 48 56](https://github.com/tubearchivist/tubearchivist/assets/88144121/82b980a5-8b8c-4eb7-98fc-bc96d865eea0)

## Expected behavior
The playlist becomes available for download

### Relevant log output

```shell
[2023-10-01 18:44:53,427: INFO/MainProcess] mingle: searching for neighbors
-- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
--- ***** ----- 
 -------------- [queues]
                .> celery           exchange=celery(direct) key=celery
                
[tasks]
  . check_reindex
  . download_pending
  . extract_download
  . index_playlists
  . manual_import
  . rescan_filesystem
  . restore_backup
  . resync_thumbs
  . run_backup
  . subscribe_to
  . thumbnail_check
  . update_subscribed
  . version_check
[2023-10-01 18:44:54,438: INFO/MainProcess] mingle: all alone
[2023-10-01 18:44:54,458: INFO/MainProcess] celery@tubearchivist ready.
[agg][primary] took 7 ms to process
[agg][watch_progress] took 4 ms to process
[agg][videos_last_week] took 3 ms to process
[agg][channel_stats] took 4 ms to process
[agg][primary] took 8 ms to process
[agg][watch_progress] took 22 ms to process
[agg][videos_last_week] took 7 ms to process
[agg][channel_stats] took 2 ms to process
https://www.youtube.com/playlist?list=PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF
[2023-10-01 18:48:43,047: INFO/MainProcess] Task subscribe_to[ad128834-35c4-415f-a27f-eed35f46a50f] received
[2023-10-01 18:48:43,048: WARNING/ForkPoolWorker-2] subscribe_to create callback
[2023-10-01 18:48:43,178: WARNING/ForkPoolWorker-2] PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF: get metadata from es
[2023-10-01 18:48:43,185: WARNING/ForkPoolWorker-2] {"_index":"ta_playlist","_id":"PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF","found":false}
[2023-10-01 18:48:43,186: WARNING/ForkPoolWorker-2] PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF: get metadata from youtube
[2023-10-01 18:48:43,598: WARNING/ForkPoolWorker-2] WARNING: [youtube:tab] Preferring "ru" translated fields. Note that some metadata extraction may fail or be incorrect.
[2023-10-01 18:48:46,188: WARNING/ForkPoolWorker-2] WARNING: [youtube:tab] Incomplete data received. Retrying (1/3)...
[2023-10-01 18:48:46,894: WARNING/ForkPoolWorker-2] WARNING: [youtube:tab] Incomplete data received. Retrying (2/3)...
[2023-10-01 18:48:47,589: WARNING/ForkPoolWorker-2] WARNING: [youtube:tab] Incomplete data received. Retrying (3/3)...
[2023-10-01 18:48:48,270: WARNING/ForkPoolWorker-2] ERROR: Incomplete data received
[2023-10-01 18:48:48,271: WARNING/ForkPoolWorker-2] https://www.youtube.com/playlist?list=PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF: failed to get info from youtube with message ERROR: Incomplete data received
[2023-10-01 18:48:48,307: WARNING/ForkPoolWorker-2] ad128834-35c4-415f-a27f-eed35f46a50f Failed callback
[2023-10-01 18:48:48,310: ERROR/ForkPoolWorker-2] Task subscribe_to[ad128834-35c4-415f-a27f-eed35f46a50f] raised unexpected: TypeError("'bool' object does not support item assignment")
Traceback (most recent call last):
  File "/root/.local/lib/python3.11/site-packages/celery/app/trace.py", line 477, in trace_task
    R = retval = fun(*args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^
  File "/root/.local/lib/python3.11/site-packages/celery/app/trace.py", line 760, in __protected_call__
    return self.run(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/home/tasks.py", line 351, in subscribe_to
    SubscriptionHandler(url_str, task=self).subscribe(expected_type)
  File "/app/home/src/download/subscriptions.py", line 342, in subscribe
    self.subscribe_type(item, expected_type=expected_type)
  File "/app/home/src/download/subscriptions.py", line 352, in subscribe_type
    PlaylistSubscription().process_url_str([item])
  File "/app/home/src/download/subscriptions.py", line 190, in process_url_str
    playlist_h.json_data["playlist_subscribed"] = subscribed
    ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'bool' object does not support item assignment
[2023-10-01 18:48:48,311: WARNING/ForkPoolWorker-2] ad128834-35c4-415f-a27f-eed35f46a50f return callback
https://www.youtube.com/playlist?list=PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF
[2023-10-01 18:55:27,693: INFO/MainProcess] Task subscribe_to[8ff01d3e-6876-48af-8d9c-d6f07439fd57] received
[2023-10-01 18:55:27,694: WARNING/ForkPoolWorker-2] subscribe_to create callback
[2023-10-01 18:55:27,815: WARNING/ForkPoolWorker-2] PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF: get metadata from es
[2023-10-01 18:55:27,820: WARNING/ForkPoolWorker-2] {"_index":"ta_playlist","_id":"PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF","found":false}
[2023-10-01 18:55:27,821: WARNING/ForkPoolWorker-2] PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF: get metadata from youtube
[2023-10-01 18:55:27,971: WARNING/ForkPoolWorker-2] WARNING: [youtube:tab] Preferring "ru" translated fields. Note that some metadata extraction may fail or be incorrect.
[2023-10-01 18:55:30,442: WARNING/ForkPoolWorker-2] WARNING: [youtube:tab] Incomplete data received. Retrying (1/3)...
[2023-10-01 18:55:31,122: WARNING/ForkPoolWorker-2] WARNING: [youtube:tab] Incomplete data received. Retrying (2/3)...
[2023-10-01 18:55:31,808: WARNING/ForkPoolWorker-2] WARNING: [youtube:tab] Incomplete data received. Retrying (3/3)...
[2023-10-01 18:55:32,525: WARNING/ForkPoolWorker-2] ERROR: Incomplete data received
[2023-10-01 18:55:32,525: WARNING/ForkPoolWorker-2] https://www.youtube.com/playlist?list=PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF: failed to get info from youtube with message ERROR: Incomplete data received
[2023-10-01 18:55:32,529: WARNING/ForkPoolWorker-2] 8ff01d3e-6876-48af-8d9c-d6f07439fd57 Failed callback
[2023-10-01 18:55:32,532: ERROR/ForkPoolWorker-2] Task subscribe_to[8ff01d3e-6876-48af-8d9c-d6f07439fd57] raised unexpected: TypeError("'bool' object does not support item assignment")
Traceback (most recent call last):
  File "/root/.local/lib/python3.11/site-packages/celery/app/trace.py", line 477, in trace_task
    R = retval = fun(*args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^
  File "/root/.local/lib/python3.11/site-packages/celery/app/trace.py", line 760, in __protected_call__
    return self.run(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/home/tasks.py", line 351, in subscribe_to
    SubscriptionHandler(url_str, task=self).subscribe(expected_type)
  File "/app/home/src/download/subscriptions.py", line 342, in subscribe
    self.subscribe_type(item, expected_type=expected_type)
  File "/app/home/src/download/subscriptions.py", line 352, in subscribe_type
    PlaylistSubscription().process_url_str([item])
  File "/app/home/src/download/subscriptions.py", line 190, in process_url_str
    playlist_h.json_data["playlist_subscribed"] = subscribed
    ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'bool' object does not support item assignment
[2023-10-01 18:55:32,533: WARNING/ForkPoolWorker-2] 8ff01d3e-6876-48af-8d9c-d6f07439fd57 return callback
[2023-10-01 19:00:00,003: INFO/MainProcess] Scheduler: Sending due task schedule_download_pending (download_pending)
[2023-10-01 19:00:00,014: INFO/MainProcess] Task download_pending[c1c53006-72c6-4a63-b065-055e73c7f0ef] received
[2023-10-01 19:00:00,016: WARNING/ForkPoolWorker-2] download_pending create callback
[2023-10-01 19:00:00,097: WARNING/ForkPoolWorker-2] M7m3A68CTJg: Downloading video
```

### Anything else?

I had previously reinstalled Tube Archivist and this feature worked correctly. But I had to reinstall the container from scratch and this error appeared. I have tried installing using the instructions from https://docs.tubearchivist.com/installation/synology/ and I have tried installing via Docker Compose, the result is the same

**Comment by bbilly1** [comment]
consolidate all yt-dlp errors in #522.

**Comment by LeoPadre** [comment]
> consolidate all yt-dlp errors in #522.

The current version of yt-dlp is used.

`Available version: stable@2023.09.24, Current version: stable@2023.09.24`
![Снимок экрана 2023-10-01 в 22 04 50](https://github.com/tubearchivist/tubearchivist/assets/88144121/3ad4b464-4a81-4199-b46b-0df330351881)

In addition, before reinstalling the container two days ago, this functionality worked correctly with the same version. What could have gone wrong after reinstalling the container? I cleaned all the folders of the container before installing.
